.drone.starlark

```diff
@@ -258,6 +274,7 @@ def acceptance():
 							'services':
 								phoenixService() +
 								owncloudService() +
+								owncloudFederatedService(suite) +
```

Comment:
You could just call this if federatedServerNeeded, maybe code like this will work, similar to other app starlark code:
```suggestion
								(owncloudFederatedService(suite) if params['federatedServerNeeded'] else []) +
```